Tasks/Common/MIGRATION.md

```diff
@@ -35,3 +35,4 @@ In addition to that, central repository usage will help us to maintain versionin
 | Package                 | Remove package folder                                               | Replace package in tasks                                            |
 | ----------------------- | ------------------------------------------------------------------- | ------------------------------------------------------------------- |
 | utility-common-v2       | [PR](https://github.com/microsoft/azure-pipelines-tasks/pull/17730) | [PR](https://github.com/microsoft/azure-pipelines-tasks/pull/16997) |
+| utility-common          | [PR](https://github.com/microsoft/azure-pipelines-tasks/pull/17737) | [PR](https://github.com/microsoft/azure-pipelines-tasks/pull/17734) |
```

Comment:
Could you please delete "remove" PR from here and add it to #17737 ?

Reply:
a removed line from the current PR will add the link on https://github.com/microsoft/azure-pipelines-tasks/pull/17737 when this PR will be merged. Thanks!